packages/react/src/components/Authenticator/shared/ValidationErrors.tsx

```diff
@@ -0,0 +1,22 @@
+import { translate } from '@aws-amplify/ui';
+import { View } from '../../../primitives/View';
+import { Text } from '../../../primitives/Text';
+
+export interface ValidationErrorsProps {
+  errors: string[];
+}
+export const ValidationErrors = ({ errors }: ValidationErrorsProps) => {
+  if (!errors || errors.length === 0) return null;
+
+  return (
+    <View data-amplify-sign-up-errors>
+      {errors.map((error, idx) => {
+        return (
+          <Text key={idx} role="alert" variation="error">
```

Comment:
Not certain it would be really be an issue here, but the value used for `key` should be unique and static to keep the reference the same between renders. Using index values for `keys` is potentially problematic as they can change if the length of the array changes

Reply:
```suggestion
          <Text key={error} role="alert" variation="error">
```

Reply:
Think that is good for now. We may need to figure something out more long term once if/when we allow users to override the values of `error` since we cannot guarantee uniqueness

Reply:
Got it.